### PR TITLE
Updated Readme

### PR DESCRIPTION
**Pull request by adithyak04**
Modified the abstract according to suggestions from previous meeting

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # KOBCC
 
 ## Abstract
-KochiOrgBook Community Cloud is a cloud interface for the city of Kochi. The Community Cloud project aims to develop a platform where participants can build, host and operate their own blockchain applications and related functions on the blockchain.
+KochiOrgBook Community Cloud is a provision given to an entity who does not have an infrastructure to build, host and run their nodes. Here, the entity can be an association like an association of lawyers, an association of doctors or it can be citizens who are trying to present their proof. It can be considered as a pool of infrastructure components on which their nodes can be implemented. The major stakeholders of the project include the citizens of Kochi, the associations who are to be onboarded and the various service providers.
+
 ## Dependent Projects
 
 The project is dependent on KOBman.
```
Comment (asa1997) [inline README.md:8]: @adithyak04 How did you come up with this?

File: README.md [hunk 2]
```diff
@@ -14,7 +15,7 @@ Decentralized Identifier or DIDs refer to a system in which there is no common s
 KOBCC will move into incubation, if approved.
 
 ## Solution
-A cloud platform providing various services to the user can be made, which consists of services including generation of DIDs, updating DIDs and verification of them. Each of nodes once hosted can be made into a cluster with the help of Kunernetes or Raspberry Pi. The cloud is accessible only to the needy ones and remains inaccessible to normal users who have the setup to host nodes. It can be considered something similar to GoDaddy website where we can get domains ending with .com at low costs.
+A cloud platform providing various services to the users who can be made, which consists of services including generation of DIDs, updating DIDs and verification of them. Each of nodes once hosted can be made into a cluster with the help of Kunernetes or Raspberry Pi. The cloud is accessible only to the needy ones and remains inaccessible to normal users who have the setup to host the nodes. It can be considered something similar to GoDaddy website where we can get domains ending with .com at low costs. Once the project is ready, each of the data will be stored in a verifiable credential registry.
```
Comment (asa1997) [inline README.md:18]: "Kunernetes" - Theres a typo

**Comment by asa1997** [comment]
@adithyak04 Merging this. Please do address the comments.

File: README.md (hunk 1 above)
Comment (adithyak04) [reply README.md:8]: @asa1997 It was from the previous corrections made by Vinod Sir

**Pull request merged**